cafe/plugins/http/cafe/engine/http/client.py

```diff
@@ -105,7 +105,8 @@ def _wrapper(*args, **kwargs):
                 log.error("An exception occured durring logging")
                 log.exception()
 
-            response_header = '\n{0}\nRESPONSE RECEIVED\n{0}\n'.format('-' * 17)
+            response_header = '\n{0}\nRESPONSE RECEIVED\n{0}\n'.format(
```

Comment:
Because the gate jobs don't have PEP8 checks yet, I can separate this, but it seems silly to do so. I would expect we want every submission to be PEP8 compliant regardless of it not being gated.